### DISCUSSION TITLE
Overriding specific accessors in properties.

### DISCUSSION THREAD

**Discussion by osamakawish** [dotnet/csharplang#5443] [Language Ideas] [answered]
Basically if I defined an abstract class like this:

```c#
public abstract class I { public abstract int Prop { get; } }
```
And I try to implement it, like this:

```c#
public class A : I { public override int Prop { get; set; } }
```
It tells me I can't do that since the setter isn't supposed to be overridden. So I'm guessing I have to define a separate variable/property/method for the setter.

But rather than having to define the setter separately, I'd prefer to keep the property, but with a setter, like this:

```c#
public class A : I { public int Prop { override get; set; } }
```

However, the above isn't valid code in C#, thus this discussion post.

Maybe it's my understanding of the language that may be lacking here though, and it should already be possible to do this in a similar way? Or is there a good reason to not allow this?

A similar issue arises with records.

**Comment by 333fred** [comment, marked as answer]
Duplicate of https://github.com/dotnet/csharplang/issues/1568.
